src/lando/main/models/uplift.py

```diff
@@ -139,3 +144,84 @@ class UpliftRevision(BaseModel):
 
     class Meta:
         unique_together = ("assessment", "revision_id")
+
+
+class MultiTrainUpliftRequest(BaseModel):
+    """Represents a single uplift request submission.
+
+    Ties together all associated uplift jobs and the uplift request
+    assessment form.
+    """
+
+    # User who requested the uplift.
+    user = models.ForeignKey(User, on_delete=models.DO_NOTHING)
+
+    # The revision IDs to be uplifted.
+    requested_revisions = models.JSONField(default=list)
+
+    assessment = models.ForeignKey(
+        UpliftAssessment,
+        on_delete=models.PROTECT,
+        related_name="uplift_request",
+    )
+
+
+class RevisionUpliftJob(BaseModel):
+    """Through model to map revisions to uplift jobs."""
+
+    uplift_job = models.ForeignKey("UpliftJob", on_delete=models.SET_NULL, null=True)
+    revision = models.ForeignKey(Revision, on_delete=models.SET_NULL, null=True)
+    index = models.IntegerField(null=True, blank=True)
```

Comment:
Could we use a `ManyToManyField` for this? https://docs.djangoproject.com/en/5.2/ref/models/fields/#manytomanyfield